hotsos/plugin_extensions/openstack/sunbeam.py

```diff
@@ -0,0 +1,24 @@
+from hotsos.core.plugins.openstack.common import (
+    OpenstackBase,
+    OpenStackChecks,
+)
+from hotsos.core.plugins.openstack.sunbeam import SunbeamInfo
+from hotsos.core.plugintools import (
+    summary_entry,
+    get_min_available_entry_index,
+)
+
+
+class SunbeamStatus(OpenstackBase, OpenStackChecks):
+    """ Get information from Sunbeam to display. """
+    summary_part_index = 14
+
+    @staticmethod
+    @summary_entry('sunbeam', get_min_available_entry_index() + 10)
```

Comment:
why the + 10 ? Although I see 1-9 are used elsewhere so just +1 i guess

Reply:
correct.